common/scala/src/main/resources/application.conf

```diff
@@ -97,8 +97,17 @@ kamon {
         }
     }
 }
-
+akka {
+    # required to allow longer shutdown-termination-limit below
+    coordinated-shutdown.default-phase-timeout = 61 s
+}
 whisk {
+    http {
+        shutdown-unready-delay = 15 seconds # /ready will return 500 for this amount of time before connection draining starts
```

Comment:
It seems it would wait for 15 seconds, and start the termination phase.
But if there are still client connections connected, are they getting connection closed error?

Reply:
This only allows an unready stage, which only affects the added /ready endpoint to return 503 (no other endpoints affected). This does not affect how akka http handles existing client connections during shutdown, so if akka allows gracefully terminating in flight requests, then no connection closed errors, but no new connections allowed. The purpose here is to allow external systems (e.g. kubernetes or reverse proxies) to probe the service and stop routing to this instance for some time before akka shutdown initiates.